tests/test_dealer.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+
+from card import Card
+from player.dealer import Dealer
+
+
+class PlayerTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.dealer = Dealer()
+
+    def assert_dealer_choice(self, cards, choose_to_stay):
```

Comment:
Lets change these arguments to `self, cards, *, choose_to_stay)`. That will make it so in order to use this, you'll need to do `self.assert_dealer_choice([1,2,3], choose_to_stay=False)`